release.yml

```diff
@@ -0,0 +1,123 @@
+# JIRA Project code where the release issue should be created
+jiraProject:     SB
+
+# If already created, provide the issue number of the Release, otherwise keep it empty.
+# It will be used by the command create-component to link the stakeholder issues
+jiraKey:         # SB-1484
```

Comment:
That key should probably be omitted if unused rather than kept empty as it's error-prone to do so, imo.

Reply:
This key will be used by the java tool when post cloning of a jira release we will create all the stakeholders jira issues. As this is only needed internally, it can be omitted from this file and just included manually when we run the command to create all component/starters issues